index.php

```diff
@@ -1,6 +1,33 @@
 <?php
-use ListingsApp\Classes\Example;
-require_once 'vendor/autoload.php';
 
-$a = new Example();
-$a->hello();
+
+namespace ListingsApp\Classes;
+
+use PDO;
+
+/** connect to database
+ * @return PDO
+ */
+function connectDB():PDO
+{
+    return new PDO('mysql:host=db;dbname=listings-feed', 'root', 'password');
+}
+
+?>
+<html>
+<body>
+<a href="index.php?type=1"><button>Sales</button></a>
```

Comment:
Best not to use a button tag inside the A - this will probably give you tab issues.

Better to style the A to look like a button instead